docs/source/add_new_model.mdx

```diff
@@ -793,9 +793,21 @@ You have now finished the coding part, congratulation! 🎉 You are Awesome! 
 **12. Upload the models to the model hub**
 
 In this final part, you should convert and upload all checkpoints to the model hub and add a model card for each
-uploaded model checkpoint. You should work alongside the Hugging Face team here to decide on a fitting name for each
+uploaded model checkpoint. You can get familiar with the hub functionalities by reading at our [Model sharing and uploading Page](https://huggingface.co/docs/transformers/model_sharing). You should work alongside the Hugging Face team here to decide on a fitting name for each
 checkpoint and to get the required access rights to be able to upload the model under the author's organization of
-*brand_new_bert*.
+*brand_new_bert*. The `push_to_hub` method, present in all models in `transformers`, is a quick and efficient way to push your checkpoint to the hub. A little snippet is pasted below:
+
+
+```python
+from pathlib import Path
+
+brand_new_bert.push_to_hub(
+    repo_path_or_name=Path("/path/to/checkpoint/"),
+    organization="<YOUR_USERNAME>",
+    commit_message="Add model",
+    use_temp_dir=True,
+)
+```
```

Comment:
I can be beneficial here to mention that the repo_path_or_name should be a path outside of the Transformers repo, otherwise it creates git repos (for the models) inside the git repo (Transformers).

Also, maybe we can mention that the same can be done for uploading the tokenizer/feature extractor/processor files to the hub.

cc @sgugger

Reply:
Makes sense, is there a sintax for tips and warnings that I may use?

Reply:
It's irrelevant here since the `use_temp_dir=True` option is selected, so it will be pushed from a temp dir.

Reply:
Feel free to resolve it, thanks :)